tools/mo/openvino/tools/mo/middle/SSliceComplex.py

```diff
@@ -109,7 +109,10 @@ def replace_pattern(self, graph: Graph, match: Dict[str, Node]):
 
         for dst in complex_node.out_port(0).get_connection().get_destinations():
             after_complex_node = dst.node
-            after_complex_node['input_rank_changed'] = True
+            if after_complex_node.type == 'Roll':
+                add_constant_to_negative_values(after_complex_node, 2, int64_array(emulated_complex_tensor_rank))
```

Comment:
As I understand, the negative axes values could be aligned to the rank by adding (-1), but the rank is already known, so it can be normalized at once.